app/operations/beta/project_board_operations.rb

```diff
@@ -32,6 +33,11 @@ def run
 
       private
 
+      def project_labels
+        labels = @project.stories.map(&:labels).reject(&:empty?)
+        labels.join(',').split(',').uniq.join(',')
```

Comment:
What about writing this line as `labels.uniq.join(',')`? Is there an absolute necessity to join the elements and then split them again?

Reply:
Now it makes sense